MapboxNavigation/RouteMapViewController.swift

```diff
@@ -388,6 +388,35 @@ class RouteMapViewController: UIViewController {
         instructionsBannerView.maneuverView.step = routeProgress.currentLegProgress.upComingStep
     }
     
+    func updateNextBanner(routeProgress: RouteProgress) {
+        let step = routeProgress.currentLegProgress.upComingStep ?? routeProgress.currentLegProgress.currentStep
+        let nextStep = routeProgress.currentLegProgress.stepAfter(step)
+        let shouldShowNextBanner = nextStep != nil ? nextStep!.distance < 1000 : false
```

Comment:
Let's use `RouteControllerHighAlertInterval` as the constant here.  https://github.com/mapbox/mapbox-navigation- ios/blob/master/MapboxCoreNavigation/Constants.swift#L95

(Maybe we should rename this 😬 )

Reply:
Can we reuse `RouteControllerMinimumDistanceForContinueInstruction` ?

Reply:
This is a little different. I think we will end up using this UI for continue instructions, but for a first pass to make this sync well with the `then`'s coming from the server we should only show this UI when linking instructions with `then`.